informaten.com.hosting.json

```diff
@@ -0,0 +1,85 @@
+{
+  "providerId": "informaten.com",
+  "providerName": "Informaten",
+  "serviceId": "hosting",
+  "serviceName": "Webhosting on Informaten.com",
+  "version": 1,
+  "syncPubKeyDomain": "informaten.com",
+  "logoUrl": "https://informaten.com/assets/logo_dark.png",
+  "description": "Enables a domain to work with Informaten webhosting",
+  "variableDescription": "ip and ip6 are the v4/v6 address of the webspace server, domain is the domain this template is applied to and hostname is the hostname of the webspace server",
+  "records": [
+    {
+      "type": "A",
+      "host": "@",
+      "pointsTo": "%ip%",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "AAAA",
+      "host": "@",
+      "pointsTo": "%ip6%",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "@",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mail.%domain%",
```

Comment:
sure about it? %domain% will resolve to the domain the template is applied to. Without trailing dot it would generate a subdomain with the same name.
If it should just be a host relative to the application of the template - no matter if subdomain or a domain (and I see the "mail" host below with an IP) then this one should be only "mail".